framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -712,6 +713,11 @@ public static void parseJdkFileAsStub(
       AnnotatedTypeFactory atypeFactory,
       ProcessingEnvironment processingEnv,
       AnnotationFileAnnotations stubAnnos) {
+    if (processingEnv.getOptions().containsKey("stubDebug")) {
+      System.out.printf(
```

Comment:
Is there a reason this is using `System.out.printf` and the other stubDebug methods are using  `.getMessager().printMessage(...)`?

Reply:
I added a comment to explain.

Reply:
What about using `stubDebugStatic`?

Reply:
That is better!